### ISSUE TITLE
[Enhancement] Home Assistant component

### ISSUE THREAD

**Issue by GorkaMM** [rgerganov/py-air-control#15]
It would be really great to have this script turned into a Home Assistant component as there is a huge community there and it would make it very easy to setup and automate Philips Air Purifiers in the Smart Home environment.

https://www.home-assistant.io

**Comment by ekeih** [comment]
Agreed! :)

* https://community.home-assistant.io/t/philips-air-purifier/53030
* https://community.home-assistant.io/t/philips-air-purifier-climate-fan-component/109166

The proper way to do this would be to move the functionality of py-air-control into its own library. Then the CLI and the Home Assistant component could use the library.

**Comment by r41d** [comment]
Could be, but since __name__ is checked in py-air-control it can also directly be used I suppose

**Comment by rgerganov** [comment]
> Could be, but since `__name__` is checked in py-air-control it can also directly be used I suppose

Exactly. There is nothing that prevents you from importing the `airctrl` package and using the `AirClient` class in 3rd party solutions.

**Comment by c-kr** [comment]
I've done this for a nagios compatible monitoring plugin. Maybe this helpes someone else to integrate in other monitoring / smart home tools: 

 https://github.com/c-kr/check_air_purifier

**Comment by mdrobniu** [comment]
I'm up for this. Somebody do something :)

**Comment by jmtatsch** [comment]
There is this https://github.com/syreel/philips-airpurifier
